cmd/standalone/main.go

```diff
@@ -55,22 +57,25 @@ func main() {
 	metricsFactory := xkit.Wrap("jaeger-standalone", expvar.NewFactory(10))
 	memStore := memory.NewStore()
 
-	builder := agentApp.NewBuilder()
-	builder.Bind(flag.CommandLine)
+	flags := &flag.FlagSet{}
+	agentApp.AddFlags(flags)
+	pMetrics.AddFlags(flags)
+	flags.Parse(os.Args)
 	flag.Parse()
 
 	runtime.GOMAXPROCS(runtime.NumCPU())
 
-	startAgent(logger, metricsFactory, builder)
+	startAgent(logger, metricsFactory)
 	startCollector(logger, metricsFactory, memStore)
 	startQuery(logger, metricsFactory, memStore)
 
 	select {}
 }
 
-func startAgent(logger *zap.Logger, baseFactory metrics.Factory, builder *agentApp.Builder) {
+func startAgent(logger *zap.Logger, baseFactory metrics.Factory) {
 	metricsFactory := baseFactory.Namespace("jaeger-agent", nil)
 
+	builder := &agentApp.Builder{}
```

Comment:
I think this is missing a call to `InitFromViper`, since the builder now has no processors defined.